src/components/filmPopups/storeMovie/sagas.ts

```diff
@@ -0,0 +1,72 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+import {DELETE_MOVIE_DATA, SEND_NEW_MOVIE_DATA, UPDATE_MOVIE_DATA} from './types';
+import {baseUrl} from "../../../url";
+import {
+  deleteMovieDataFailure,
+  deleteMovieDataRequest, deleteMovieDataSuccess,
+  sendNewMovieDataFailure,
+  sendNewMovieDataRequest,
+  sendNewMovieDataSuccess, updateMovieDataFailure, updateMovieDataRequest, updateMovieDataSuccess
+} from "./actions";
+import axios from "axios";
+import { showAlert } from '../storeAlerts/actions';
+
+
+function* sendNewMovie(data:any) {
+  try {
+    yield put(sendNewMovieDataRequest());
+    const res = yield call(() => axios.post(`${baseUrl}movie`, {...data.payload}));
```

Comment:
This should result in:
> 'yield' expression implicitly results in an 'any' type because its containing generator lacks a return-type annotation.ts(7057)
Which might be solved by strong-typing result of the `yield`:
```ts
const res: IMovie = yield call(fetch, baseUrl)
```